open-api/rest-catalog-open-api.yaml

```diff
@@ -3858,8 +3924,8 @@ components:
           items:
             $ref: '#/components/schemas/PrimitiveTypeValue'
           description:
-            "A list of partition field values ordered based on the fields of the partition spec specified by the 
-            `spec-id`"
+            A list of partition field values ordered based on the fields of
+            the partition spec specified by the `spec-id`
```

Comment:
I think this change was accidental?

Reply:
I can roll it back, but the intent was to fix the line wrapping here.